interp/interp_test.go

```
@@ -2629,6 +2629,49 @@ func TestInterpreter_Run(t *testing.T) {
 	})
 
+	t.Run("parity/host struct field write reaches the Go value", func(t *testing.T) {
+		type counter struct {
+			Count  int32
+			hidden int32
+		}
+		bump := func(c *counter) int32 { c.hidden++; return c.Count }
+
+		runHost := func(opts ...func(*option)) (types.Value, int32) {
+			setup := New(program.New(nil))
+			defer setup.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `setup.Close` is not checked

(errcheck)